app/src/main/java/org/oppia/android/app/home/HomeFragmentPresenter.kt

```diff
@@ -72,7 +76,7 @@ class HomeFragmentPresenter @Inject constructor(
     homeLayoutManager.spanSizeLookup = object : GridLayoutManager.SpanSizeLookup() {
       override fun getSpanSize(position: Int): Int {
         return if (position < homeAdapter.itemCount &&
-          homeAdapter.getItemViewType(position) === ViewType.TOPIC_LIST.ordinal
+          homeAdapter.getItemViewType(position) == ViewType.TOPIC_LIST.ordinal
```

Comment:
seems like a nice catch. 
We don't need a Referential Equality(===) here.

Reply:
Thanks.